index.js

```diff
@@ -1,18 +1,17 @@
 var quadraticElement = document.getElementById("quadratic");
 var solutionElement = document.getElementById("solution");
-// (ax + b)(cx + d)
-var fA, fB, fC, fD;
 
-// ax^2 + bx + c
+var fA, fB, fC, fD;
 var qA, qB, qC;
 
 var url = "https://latex.codecogs.com/gif.latex?\\huge&space;";
 
+function getRandomNums() {
+  return Math.ceil(Math.random()*5);
+}
+
 function generate(event){
-    fA = Math.ceil(Math.random()*5);
-    fB = Math.ceil(Math.random()*5);
-    fC = Math.ceil(Math.random()*5);
-    fD = Math.ceil(Math.random()*5);
+    [fA, fB, fC, fD] = [getRandomNums(), getRandomNums(), getRandomNums(), getRandomNums()];
```

Comment:
In my opinion having the assignments on separate lines is more readable

Reply:
in my opinion you should be quiet